content/tracking/2-mobile_trackers_usage.md

```diff
@@ -44,6 +44,26 @@ TrackerController tracker = Snowplow.createTracker(context,
 );
 ```
 
+{{% /tab %}}
+
+{{% tab name="React Native" %}}
+
+```typescript
+import { createTracker } from '@snowplow/react-native-tracker';
+
+const tracker = createTracker(
+    'appTracker',
+    {
+      endpoint: COLLECTOR_URL,
```

Comment:
trailing comma. It's correct in typescript but I've seen you haven't used it later so don't know if you prefer to be consistent.

Reply:
Thanks, added trailing commas in all typescript snippets as I think that is the convention.